src/xtt.c

```diff
@@ -237,13 +227,34 @@ enftun_xtt_handshake(const char** server_hosts,
     }
 
     // 2) Make TCP connection to server.
-    ret = sock->ops.connect_any(sock, server_hosts, (char*) server_port);
+    enftun_tcp_native_init(&sock);
+    ret = sock.ops.connect_any(&sock, server_hosts, (char*) server_port, mark);
+    enftun_log_debug("WiFi %s\n", (ret<0 ? "Failed " : "Succeeded"));
```

Comment:
`enftun` is used is more places that the router card, so the debug comment shouldn't start with "WiFi".  It should start with something like "TCP" or "TCP native".